<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w</class>
 <widget class="QWidget" name="w">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(images.jfif);
</string>
  </property>
  <widget class="QFrame" name="frS">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>10</y>
     <width>1471</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frS{
	border-image: url(im3.jpeg) 0 0 0 0 stretch stretch;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="cin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nom">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pr">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="age">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ad">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="nt">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>TUNISIENNE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ANGLAISE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bahrein</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bresil</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>canada</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>chine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>colombie</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nouvel élément</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ALLEMANDE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RWANDAISE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ESPAGNALE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ALGERIENNE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FRANCAISE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RUSSE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JAPONAISE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>INDIENNE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MAROCAINE</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="tel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="jour">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mois">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="annee">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>410</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;Annee de linfection&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; font-style:italic; text-decoration: underline; color:#333b68;&quot;&gt;ajouter personne&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>29</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;CIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;Nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(77, 179, 200);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;Prenom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;Age&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;Adresse&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;Nationalite&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;Telephone&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;Date de linfection:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;Jour de l'infection&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;Mois de l'infection&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#333b68;&quot;&gt;Decede&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btr1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btr2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ajouter">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../test/rec.qrc"/>
 </resources>
 <connections/>
</ui>
